.travis.yml

```diff
@@ -6,20 +6,26 @@ branches:
     - v0.6
     - v0.6-PHP_5.2
     - documentation
-services: redis-server
+before_install:
+  - docker run -d --rm -p 127.0.0.1:6379:6379 redis:3
```

Comment:
good idea. this might make it easier to test against different versions of redis